ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -725,6 +725,7 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                          gru_cell->get_linear_before_reset());
             break;
         }
+        case OP_TYPEID::LSTMCell_v0:
```

Comment:
Is it correct that we apply static cast to LSTMCell v4 for LSTMCell v0?

Reply:
I will fix it in the next PR

Reply:
ticket 43220